### DISCUSSION TITLE
Building DEBs inside mkosi (analogous to "Building RPMs from source with mkosi")?

### DISCUSSION THREAD

**Discussion by septatrix** [systemd/mkosi#2701] [Q&A]
Is someone using mkosi to (re)build debian packages? It would be great to have a documentation for that similar to the doc for building RPMs from source.

Currently I am trying to rebuild systemd for bookworm with the sd-sysupdate component enabled. I will manage to hack something together build but it would be great to have this process documented for future users, ideally with flags similar to the rpmbuild invokation to prevent polluting the image etc.

**Comment by behrmann** [comment]
I haven't looked at it yet, but I think the new integration tests in the systemd repo, that use mkosi, also produce RPM and deb packages.

Otherwise, depending on your needs installing fpm and using that to generate a deb might be the easiest way.

**Reply by daandemeyer** [reply]
Just look at the systemd repo, we rebuild debs there. Will review a PR to document that in mkosi like we do for rpms.
